Header header
std_msgs/Int32 sensor_id
evarobot_adp/Object[] objects
